src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/SourceEditModePageObject.java

```diff
@@ -331,8 +340,8 @@ public void verifyVideoWidth(int widthDesired) {
         content.substring(content.indexOf("px") - 4, content.indexOf("px") - 1)
     );
     Assertion.assertNumber(
-            widthDesired, width,
-            "width is " + width + " should be " + widthDesired
+        widthDesired, width,
+        "width is " + width + " should be " + widthDesired
```

Comment:
width what? ;-)
Again, I know that's a formatting change only. Let's use this as an opportunity to make this piece of code better. Even if it's as simple as 'Video thumbnail width in a portable infobox is X, should be Y to fill the space fully without cropping the thumbnail'